test/OpenIddict.Server.AspNetCore.IntegrationTests/OpenIddictServerAspNetCoreIntegrationTests.cs

```diff
@@ -604,7 +604,7 @@ void ConfigurePipeline(IApplicationBuilder app)
                         await context.Response.WriteAsync(JsonSerializer.Serialize(
                             new OpenIddictResponse(result.Principal.Claims.GroupBy(claim => claim.Type)
                                 .Select(group => new KeyValuePair<string, string[]>(
-                                    group.Key, group.Select(claim => claim.Value).ToArray())))));
+                                    group.Key, group.Select(claim => claim.Value).ToArray()))!)));
```

Comment:
🤷

Reply:
I took a second look and just and there is a much more reasonable solution: Align the types :)